core/src/main/java/com/novoda/noplayer/internal/exoplayer/NoPlayerAdsLoader.java

```diff
@@ -32,6 +33,8 @@
     private AdvertsLoader.Cancellable loadingAds;
 
     private Optional<NoPlayer.AdvertListener> advertListener = Optional.absent();
+    @Nullable
+    private List<AdvertBreak> advertBreaks;
```

Comment:
I guess you already thought of this but, can we have an empty list instead?

Reply:
Yes we probably can 😄

Reply:
https://github.com/novoda/no-player/pull/208/commits/7ddefbc2641359663aa5fcde7836be5e3c08ad31